src/Commands/JobMakeCommand.php

```diff
@@ -79,7 +80,14 @@ public function getArguments()
     {
         return [
             ['job', InputArgument::REQUIRED, 'The job\'s name.'],
-            ['domain', InputArgument::REQUIRED, 'The domain to be responsible for the job.'],
+            ['domain', InputArgument::OPTIONAL, 'The domain to be responsible for the job.'],
```

Comment:
a domain is always required for generating jobs, it can't be generated elsewhere.